index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]>      <html class="no-js"> <![endif]-->
+<html lang="fr">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Portfolio: setup and mobile version skeleton</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet">
+</head>
+
+<body>
+    <header class="navbar">
+        <a class="logo">My Logo</a>
+        <nav>
+            <ul class="nav-links">
+                <li><label id="checkbox_toggle" class="hamburger">&#9776;</label></li>
+            </ul>
```

Comment:
##OPTIONAL

- [ ] Kindly remove the list-style property from the hamburger menu icon 👍🏿 
![dot](https://user-images.githubusercontent.com/69638013/159010955-8a7a9635-dd68-498a-bfcd-5411de477c46.png)